src/node_dotenv.cc

```diff
@@ -154,6 +215,33 @@ void Dotenv::ParseContent(const std::string_view input) {
       continue;
     }
 
+    if (content.front() == '[') {
```

Comment:
Can you add a comment to here explaining why you're checking for [?

Reply:
Yes, but a year from now, a contributor might have some hard time understanding the reasoning.

Reply:
comment added :slightly_smiling_face: